charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         release: {{ .Release.Name }}
         timescaleNodeType: access
         {{ with .Values.accessNode.extraPodLabels -}}
```

Comment:
```suggestion
        {{- with .Values.accessNode.extraPodLabels }}
```
`-` should be on the left side, not on the right to reduce confusion
that's probably the reason this odd thing is working

Reply:
moved the `-` but it's the same correct render

Also tried with but still renders correctly
```
       {{- if .Values.dataNode.extraPodLabels }}
       {{ toYaml .Values.dataNode.extraPodLabels }}
       {{- end }}
```

Reply:
move `-` to the left